jcl/j9modules.xml

```diff
@@ -105,6 +110,37 @@
 		</sequential>
 	</macrodef>
 
+	<macrodef name="replaceexportsoropens">
```

Comment:
It worths to note in this PR description that following macros are also in https://github.com/eclipse/openj9/pull/444 (not merged yet), they are identical.